docs/contributing.rst

```diff
@@ -29,6 +29,15 @@ Workflow is pretty straightforward:
   7. Optionally make backport Pull Request(s) for landing a bug fix
      into released aiohttp versions.
 
+.. note::
+
+   The project uses *Squash-and-Merge* strategy for *GitHub Merge* button.
+
+   Basically it means that there is **no need to rebase*** a Pull Request against
```

Comment:
`***` -> **

Reply:
Also, I'd rephrase to `you don't have to`. Because it's a person's choice. I personally like to be doing this a lot.